other/materials_designer/create_interface_with_min_strain_zsl.ipynb

```diff
@@ -46,13 +46,11 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    interfaces_range = slice(0, 1)  # select the first interface with the lowest strain
</pre><p>Let's move the comment above and add with the lowest strain and number of atoms</p><p> </p><p>The actual selection by termination and slice functionality can go to the holder class itself too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>